<PISI>
    <Source>
        <Name>python-exceptiongroup</Name>
        <Homepage>https://github.com/agronholm/exceptiongroup</Homepage>
        <Packager>
            <Name>Algent Albrahimi</Name>
            <Email>algent@protonmail.com</Email>
        </Packager>
        <License>MIT</License>
        <PartOf>programming.python</PartOf>
        <Summary xml:lang="en">Backport of PEP 654 (exception groups)</Summary>
        <Description xml:lang="en">This is a backport of the BaseExceptionGroup and ExceptionGroup classes from Python 3.11.
</Description>
        <Archive type="binary" sha1sum="79eb0752a961b8e0d15c77d298c97498fbc89c5a">https://getsol.us/sources/README.Solus</Archive>
    </Source>
    <Package>
        <Name>python-exceptiongroup</Name>
        <Summary xml:lang="en">Backport of PEP 654 (exception groups)</Summary>
        <Description xml:lang="en">This is a backport of the BaseExceptionGroup and ExceptionGroup classes from Python 3.11.
</Description>
        <PartOf>programming.python</PartOf>
        <Files>
            <Path fileType="library">/usr/lib/python3.10/site-packages/exceptiongroup-1.1.0.dist-info/LICENSE</Path>
            <Path fileType="library">/usr/lib/python3.10/site-packages/exceptiongroup-1.1.0.dist-info/METADATA</Path>
            <Path fileType="library">/usr/lib/python3.10/site-packages/exceptiongroup-1.1.0.dist-info/RECORD</Path>
            <Path fileType="library">/usr/lib/python3.10/site-packages/exceptiongroup-1.1.0.dist-info/WHEEL</Path>
            <Path fileType="library">/usr/lib/python3.10/site-packages/exceptiongroup/__init__.py</Path>
            <Path fileType="library">/usr/lib/python3.10/site-packages/exceptiongroup/__pycache__/__init__.cpython-310.opt-1.pyc</Path>
            <Path fileType="library">/usr/lib/python3.10/site-packages/exceptiongroup/__pycache__/__init__.cpython-310.pyc</Path>
            <Path fileType="library">/usr/lib/python3.10/site-packages/exceptiongroup/__pycache__/_catch.cpython-310.opt-1.pyc</Path>
            <Path fileType="library">/usr/lib/python3.10/site-packages/exceptiongroup/__pycache__/_catch.cpython-310.pyc</Path>
            <Path fileType="library">/usr/lib/python3.10/site-packages/exceptiongroup/__pycache__/_exceptions.cpython-310.opt-1.pyc</Path>
            <Path fileType="library">/usr/lib/python3.10/site-packages/exceptiongroup/__pycache__/_exceptions.cpython-310.pyc</Path>
            <Path fileType="library">/usr/lib/python3.10/site-packages/exceptiongroup/__pycache__/_formatting.cpython-310.opt-1.pyc</Path>
            <Path fileType="library">/usr/lib/python3.10/site-packages/exceptiongroup/__pycache__/_formatting.cpython-310.pyc</Path>
            <Path fileType="library">/usr/lib/python3.10/site-packages/exceptiongroup/__pycache__/_version.cpython-310.opt-1.pyc</Path>
            <Path fileType="library">/usr/lib/python3.10/site-packages/exceptiongroup/__pycache__/_version.cpython-310.pyc</Path>
            <Path fileType="library">/usr/lib/python3.10/site-packages/exceptiongroup/_catch.py</Path>
            <Path fileType="library">/usr/lib/python3.10/site-packages/exceptiongroup/_exceptions.py</Path>
            <Path fileType="library">/usr/lib/python3.10/site-packages/exceptiongroup/_formatting.py</Path>
            <Path fileType="library">/usr/lib/python3.10/site-packages/exceptiongroup/_version.py</Path>
            <Path fileType="library">/usr/lib/python3.10/site-packages/exceptiongroup/py.typed</Path>
        </Files>
    </Package>
    <History>
        <Update release="1">
            <Date>2023-01-01</Date>
            <Version>1.1.0</Version>
            <Comment>Packaging update</Comment>
            <Name>Algent Albrahimi</Name>
            <Email>algent@protonmail.com</Email>
        </Update>
    </History>
</PISI>